src/datasources/errors/http-error-factory.ts

```diff
@@ -3,21 +3,21 @@ import axios, { AxiosError } from 'axios';
 import { HttpExceptionPayload } from './interfaces/http-exception-payload.interface';
 
 /**
```

Comment:
This class is named `*ErrorFactory` but it could be `*ErrorMapper` as well. I'm open to hear name suggestions @fmrsabino 🙂